### PR TITLE
Create ru.json

### PR DESCRIPTION
**Pull request by winqooq**
Draft translation into Russian

### CODE REVIEW

**Comment by c0re100** [comment]
If #6 #5 ready to merge, ping me :)

**Comment by winqooq** [comment]
@c0re100 done :)

**Review by c0re100 — CHANGES_REQUESTED** [review]

File: ru.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "lng_rights_chat_send_games": "Отправка игр",
+    "lng_rights_chat_use_inline": "Использование инлайн-ботов",
+    
+    "lng_media_selected_gif#one": "{count} GIF",
+    "lng_media_selected_gif#few": "{count} GIF",
+    "lng_media_selected_gif#many": "{count} GIF",
+    "lng_media_selected_gif#other": "{count} GIF",
+    "lng_media_type_gifs": "GIF",
+    "lng_profile_gifs#one": "{count} GIF",
+    "lng_profile_gifs#few": "{count} GIF",
```
[the remark this answers is not in the document]
Comment (winqooq) [reply ru.json:61]: it has no translation in either Russian or Ukrainian, it is used in this form
just "GIF"

File: ru.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    "lng_admin_log_banned_send_gifs": "Отправка GIF",
+    "lng_admin_log_banned_send_games": "Отправка игр",
+    "lng_admin_log_banned_use_inline": "Использование инлайн-ботов",
+    "lng_rights_chat_send_stickers2": "Отправка стикеров",
+    "lng_rights_chat_send_gifs": "Отправка GIF",
+    "lng_rights_chat_send_games": "Отправка игр",
+    "lng_rights_chat_use_inline": "Использование инлайн-ботов",
+    
+    "lng_media_selected_gif#one": "{count} GIF",
+    "lng_media_selected_gif#few": "{count} GIF",
```
[the remark this answers is not in the document]
Comment (winqooq) [reply ru.json:56]: sorry, but without this data, this graph will look like this
Good
bad

[the remark this answers is not in the document]
Comment (c0re100) [reply ru.json:56]: Look like buggy on customize language loader, I will fix it later.
So it is safe to remove this 2 string.

File: ru.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (c0re100) [reply ru.json:61]: I mean `few` and `many`

File: ru.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "lng_media_type_gifs": "GIF",
+    "lng_profile_gifs#one": "{count} GIF",
+    "lng_profile_gifs#few": "{count} GIF",
```
Comment (c0re100) [inline ru.json:59]: deleted by mistake lol
only have #one and #other.

**Review by c0re100 — APPROVED** [review]

**Comment by c0re100** [comment]
Thank you~

**Comment by c0re100** [comment]
hmmm.......
Look like Russian can't display gif header text correctly.....
Some group have gif header, but some haven't......
I don't know why, so confused................

**Comment by winqooq** [comment]
@c0re100 because previously deleted lines (#few and #many) are required for Russian and Ukrainian languages

(you can see which variables correspond to the language here https://translations.telegram.org/?l=language%2Fnew)

look in the Kotatogram Desktop (https://github.com/kotatogram/kotatogram-desktop), maybe you can find there a solution of this problem

**Comment by c0re100** [comment]
> @c0re100 because previously deleted lines (#few and #many) are required for Russian and Ukrainian languages
> Screenshot_2020-10-22 Translations(1)
> (you can see which variables correspond to the language here https://translations.telegram.org/?l=language%2Fnew by choosing the base language)
> 
> look in the Kotatogram Desktop (https://github.com/kotatogram/kotatogram-desktop), maybe you can find there a solution of this problem

WTF is telegram doing......lol
so confusing...
I guess I can fix it on client side.

**Comment by winqooq** [comment]
@c0re100, tell me, do i have to create a new pull to update the translation?

**Comment by c0re100** [comment]
> @c0re100, tell me, do i have to create a new pull to update the translation?

No need, cause still in draft stage.
You can PR when 2.4.4.2 out.

**Comment by winqooq** [comment]
> No need, cause still in draft stage.
> You can PR when 2.4.4.2 out.

Ok, can you tell me how exactly can I update the localization

_maybe I'll just send you a ready *.json translation? :)_

**Comment by c0re100** [comment]
> > No need, cause still in draft stage.
> > You can PR when 2.4.4.2 out.
> 
> Ok, can you tell me how exactly can I update the localization
> 
> _maybe I'll just send you a ready *.json translation? :)_

If you don't know how to use git.
You can delete your repo and fork again, then create pull request with update.

**Comment by winqooq** [comment]
> You can delete your repo and fork again, then create pull request with update.

ok, got it.

what should I choose "Create pull request" or "Create draft pull request" when opening a pull request? _so that I can update the translation in the future without creating a new pull request every time_

**Pull request merged**